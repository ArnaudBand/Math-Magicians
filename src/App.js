import React from 'react';
import Header from './header/header';
import Calculator from './components/calculator';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render = () => (
    <>
      <Header />
      <Calculator />
    </>
  )
}

export default App;
