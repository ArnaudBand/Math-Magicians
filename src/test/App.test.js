import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import App from '../App';

it('App renders correctly', () => {
  // eslint-disable-next-line no-undef
  const tree = renderer
    .create(
      <MemoryRouter>
        <App />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
