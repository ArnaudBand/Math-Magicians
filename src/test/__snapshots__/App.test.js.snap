// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="calculator"
>
  <header
    className="header"
  >
    <h1
      className="logo"
    >
      Math Magicians
    </h1>
    <div
      className="right_side"
    >
      <a
        className="menu_link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="menu_link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="menu_link"
        href="/Quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </header>
  <header
    className="div_header"
  >
    <h3
      className="text_title"
    >
      Welcome to our page !
    </h3>
    <p>
      In hac habitasse platea dictumst vestibulum. Aliquet porttitor lacus luctus accumsan. Scelerisque purus semper eget duis at tellus at. In fermentum et sollicitudin ac orci phasellus egestas tellus. Cursus turpis massa tincidunt dui ut ornare. Id leo in vitae turpis massa sed elementum tempus egestas. Vestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare massa. Ultrices eros in cursus turpis massa tincidunt dui. Iaculis at erat pellentesque adipiscing commodo elit. Vitae ultricies leo integer malesuada nunc vel. Massa massa ultricies mi quis hendrerit dolor magna. Nibh tortor id aliquet lectus proin nibh nisl. Est sit amet facilisis magna etiam tempor orci eu. Ut sem viverra aliquet eget sit amet tellus. Pharetra massa massa ultricies mi quis hendrerit dolor magna.
    </p>
    <p>
      In hac habitasse platea dictumst vestibulum. Aliquet porttitor lacus luctus accumsan. Scelerisque purus semper eget duis at tellus at. In fermentum et sollicitudin ac orci phasellus egestas tellus. Cursus turpis massa tincidunt dui ut ornare. Id leo in vitae turpis massa sed elementum tempus egestas. Ve stibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare massa. Ultrices eros in cursus turpis massa tincidunt dui. Iaculis at erat pellentesque adipiscing commodo elit . Vitae ultricies leo integer malesuada nunc vel. Massa massa ultricies mi quis hendrerit dolor magna. Nibh tortor id aliquet lectus proin nibh nisl. Est sit amet facilisis magna etiam tempor orci eu. Ut sem viverra aliquet eget sit amet tellus. Pharetra massa massa ultricies mi quis hendrerit dolor magna.
    </p>
    <p
      className="text-center"
    >
      By Arnaud
    </p>
  </header>
</div>
`;
